### PR TITLE
[IA-1490] Remove hardcoded 5s proxy timeout

### PR DESCRIPTION
**Pull request by rtitle**
https://broadworkbench.atlassian.net/browse/IA-1490

cc @calbach 

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala [hunk 1]
```diff
@@ -65,6 +66,9 @@ class ProxyService(
 )(implicit val system: ActorSystem, materializer: ActorMaterializer, executionContext: ExecutionContext)
     extends LazyLogging {
 
+  final val requestTimeout = toScalaDuration(system.settings.config.getDuration("akka.http.server.request-timeout"))
```
Comment (kyuksel) [inline http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala:69]: so it'll be `60s`, correct?

Comment (rtitle) [reply http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala:69]: Yes: https://github.com/broadinstitute/firecloud-develop/blob/dev/base-configs/leonardo/leonardo.conf.ctmpl#L71

I was planning to confirm that after the auto-tests run by looking at Leo app logs in `fiab-stop`.

Comment (kyuksel) [reply http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala:69]: I first wondered if 60s for Jupyter to Proxy may be too much, in which case having a smaller timeout could alert us to an slowness issue but that's probably not the best way to detect such issues. We may want to have metrics for it if possible.

> I was planning to confirm that after the auto-tests run by looking at Leo app logs in fiab-stop.

Good idea.

**Review by kyuksel — APPROVED** [review]

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala (hunk 1 above)
Comment (rtitle) [reply http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala:69]:
```
[INFO] [15:49:33.281] [ioapp-compute-4] o.b.d.w.l.service.ProxyService - Leo proxy request timeout is 1 minute
```

**Pull request merged**